renderer/styles/style.css

```diff
@@ -0,0 +1,98 @@
+.flip {
```

Comment:
`.flip` I think this was added for the expand button in question component. This can be removed from here, as it is already added to question component